enterprise-suite/values.yaml

```diff
@@ -75,6 +75,16 @@ imageCredentials:
 #  username: setme
 #  password: setme
 
+# If true, an alertmanager will be deployed. Set to false and modify `alertManagers` list to use existing alertmanager(s).
+createAlertManager: true
+
+# List of alertmanager addresses (can be k8s local DNS service names)
+alertManagers: 
+  - alertmanager
+
+# Port to use for accessing alertmanager services
+alertManagerPort: 9093
```

Comment:
How come the port is separate from the hostname? Wouldn't it be simpler to include the port on the address?
```
alergManagers
- alertmanager:9093
- alertmanager2:9093
- alertmanager3:4000 # external
```

Reply:
Okay, I see we do this to make the implementation simpler (like the configmap reloader stuff). But I think this should be optimized for users, so it should contain the port on the list of alertmanagers and not have it be a separate var.

For the alertmanager we create, I think it's fine to hardcode 9093 on the config reloader since we tightly control it and know what port it's listening on.

Reply:
Good point, I changed `alertManagers` to include ports in each address.